bin/osg-topology

```diff
@@ -151,6 +156,11 @@ def main():
     oparser = get_parser()
 
     args = oparser.parse_args()
+    if getattr(args, "fqdn") is not None:
+        if args.fqdn:
```

Comment:
Aren't these two if statements superfluous? Can we ever reach the else block?

Reply:
Yeah, we can reach the else block if args.fqdn is False, which is what I'm testing for.